tests/python_package_test/test_arrow.py

```diff
@@ -132,7 +141,10 @@ def assert_datasets_equal(tmp_path: Path, lhs: lgb.Dataset, rhs: lgb.Dataset):
     ("arrow_table_fn", "dataset_params"),
     [  # Use lambda functions here to minimize memory consumption
         (lambda: generate_simple_arrow_table(), dummy_dataset_params()),
-        (lambda: generate_simple_arrow_table(empty_chunks=True), dummy_dataset_params()),
+        (
+            lambda: generate_simple_arrow_table(empty_chunks=True),
+            dummy_dataset_params(),
+        ),
```

Comment:
If the current state of this branch passes, don't bother reverting this... let's merge and unblock CI.

But if you do need to push another commit, could you please revert these style-only changes? They seem unrelated.

Reply:
Sorry, this was unintentional, got rid of this.